### ISSUE TITLE
Run in mobile simulation on windows

### ISSUE THREAD

**Issue by tandem97** [fyne-io/fyne#2454]
### Describe the bug:
go run -tags mobile main.go does not work on windows

**Comment by sanguohot** [comment]
Still not work on windows with the latest code.
`go run -tags mobile cmd/fyne_demo/main.go`

**Comment by Jacalz** [comment]
> Still not work on windows with the latest code.
> `go run -tags mobile cmd/fyne_demo/main.go`

That is correct. The situation should be similar or the same because very little have changed  on that front. There is a PR open for improving it (https://github.com/fyne-io/fyne/pull/2582) but it hasn’t landed.

I don’t think anyone in our team normally does development on Windows. If you are interested in this feature, you could always consider opening a PR or help sponsor us at https://fyne.io/sponsor/. There should be something about placing a bounty on a specific issue to fix but I can’t find it at the moment.
